data/50-default.json

```diff
@@ -154,5 +154,17 @@
     "ref-kind": "app",
     "name": "com.endlessm.Sidetrack",
     "branch": "stable"
+  },
+  {
+    "action": "update",
+    "serial": 2020040300,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps",
+    "remote": "eos-apps",
+    "name": "com.hack_computer.Clubhouse",
+    "branch": "eos3",
+    "filters": {
+        "architecture": ["x86_64"]
+    }
```

Comment:
I don't think you need the filter – the clubhouse won't be installed on armv7hl, so the update block will have no effect there. (And as written here it's wrong – it doesn't include 64-bit arm for raspberry pi.)

I also don't think you need the remote or collection-id. There's actually a schema for this file https://github.com/endlessm/eos-updater/blob/84fa3900dcc20cc0aa1b065a496a544e918b63d4/eos-updater-flatpak-installer/eos-updater-autoinstall.schema.json#L66-L91 – I don't know if it's enforced but remote and collection-id are not listed for update actions.

Reply:
Thanks for the review, I've just updated removing the collection-id, remote and the filters